### ISSUE TITLE
docker build returns an error

### ISSUE THREAD

**Issue by joseraeiro** [ThoughtfulDev/EagleEye#153]
Hello, I was following the instructions on the README page and when I try to run:

`sudo docker build -t eagle-eye .`

It gives the following error at some point

E: Unable to locate package python3.6
E: Couldn't find any package by glob 'python3.6'
E: Couldn't find any package by regex 'python3.6'
E: Unable to locate package python3.6-dev
E: Couldn't find any package by glob 'python3.6-dev'
E: Couldn't find any package by regex 'python3.6-dev'
The command '/bin/sh -c apt-get update && add-apt-repository -y ppa:deadsnakes/ppa && apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y python3.6 python3.6-dev' returned a non-zero code: 100

I do believe this is an issue related with the repository ppa:deadsnakes/ppa not being accessible. Not sure if this is temporary or not. I'll try later on and let you know if it worked.

**Comment by victor-souza1997** [comment]
i am having the same problem. Have you found a solution to it?

**Comment by joseraeiro** [comment]
No, not really. I guess that it's a question of searching for a substitute repository which has Python 3.6 in it.

I saw a page on Stackoverflow but neither of the solutions there worked, so I created this bug issue.

Please, do let me know if you fixed the issue!

**Comment by r4lix** [comment]
deadsnake ppa and python 3.6 are EOL, better create a new dockerfile

**Comment by jd-rk9** [comment]
I ran iinto the same issue and decided to build a more modern dockerfile this will build but won't solve other issues since the software isn't actively maintained anymore. 

FROM debian:bullseye

RUN apt-get clean && apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y locales
RUN locale-gen en_US.UTF-8
ENV LANG en_US.UTF-8
ENV LANGUAGE en_US:en
ENV LC_ALL en_US.UTF-8
ENV TERM dumb
ENV PYTHONIOENCODING=utf-8

RUN apt update &&  apt upgrade -y
RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y libgtk-3-dev libboost-all-dev build-essential cmake libffi-dev
RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y firefox-esr
RUN apt-get install -y git python3 python3-pip python3-dev


RUN git clone https://github.com/ThoughtfulDev/EagleEye
RUN cd EagleEye && pip3 install -r requirements.txt
RUN pip3 install --upgrade beautifulsoup4 html5lib spry

WORKDIR EagleEye
ADD https://github.com/mozilla/geckodriver/releases/download/v0.32.0/geckodriver-v0.32.0-linux64.tar.gz /EagleEye/geckodriver.tar.gz
RUN tar -xvf geckodriver.tar.gz
RUN mv geckodriver /usr/bin/geckodriver
RUN chmod +x /usr/bin/geckodriver
RUN rm -r /EagleEye/known/
ENTRYPOINT bash /entry.sh
